src/System.ComponentModel.Composition/src/System/ComponentModel/Composition/Diagnostics/CompositionTraceSource.cs

```diff
@@ -9,7 +9,7 @@ namespace System.ComponentModel.Composition.Diagnostics
 {
     internal static class CompositionTraceSource
     {
-#if FEATURE_TRACING
+#if  FEATURE_TRACING
```

Comment:
extra spaces here and other places

Reply:
causing a bunch of extra files in the diff

Reply:
I added this diff in a separate commit to show where there are remaining #if conditions left in the solution.

Reply:
Oh of course - I forgot. OK, you'll want to resolve those...